src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -32,6 +32,18 @@
                   Description="Specifies the base namespace for files added to the project."
                   Category="General" />
 
+  <BoolProperty Name="TargetMultipleFrameworks"
+                DisplayName="Target multiple frameworks"
+                Category="General">
+    <BoolProperty.DataSource>
+      <DataSource Persistence="ProjectFileWithInterception"
+                  HasConfigurationCondition="False" />
+    </BoolProperty.DataSource>
+    <BoolProperty.Metadata>
+      <NameValuePair Name="DependsOn" Value="TargetFrameworkMoniker;TargetFrameworks" />
```

Comment:
What do we do about dependencies across pages?

1. Assume names are unique across pages
2. Require the page be specified via some syntax (e.g. `Application::TargetFrameworkMoniker`)
3. Something else

Reply:
Do we have any examples of cross-page dependencies? Or even possible cross-page dependencies? Arguably, if there is a dependency relationship between two properties then maybe they should be on the same page.

What if we just didn't support cross-page dependencies or visibility conditions for v1?

Reply:
We have examples of this. For example, changing `AssemblyName` impacts the `Packaging` page. We cannot assume or enforce relationships existing only within a single page.

Reply:
Ah, yes. I remember that one now.

OK, I would propose `PageName:PropertyName` (with just one color) for the syntax in `DependsOn`.

Reply:
I don't feel strongly either way, but `::` is a common operator for resolving scopes (C#, Java, C++, Rust, PHP, Ruby).